styles.css

```diff
@@ -0,0 +1,122 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&display=swap');
+
+body {
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+}
+
+.container {
+  height: 500px;
+  background: white;
+  width: 375px;
+  background-repeat: no-repeat;
+  background-image:
+    url(./images/Ellipse\ 8.png),
+    url(./images/Rectangle\ 7.png),
+    url(./images/Ellipse\ 3.png);
+  background-position: center, left, right;
+}
+
+.circle {
+  background-repeat: no-repeat;
+  background-image:
+    url(./images/Ellipse\ 11.png),
+    url(./images/Rectangle\ 5.jpg);
+  background-position: left, center;
+  padding: 1em;
+}
+
+.toolbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.logo {
+  color: #fff;
+  margin: 0 0 0 0.7em;
+  font-weight: 900;
+  cursor: pointer;
+  padding: 0.5em;
+}
+
+.logo-txt {
+  color: rgb(8, 8, 8);
+  text-decoration: none;
+}
+
+.rec {
+  background-image: url(./images/Ellipse\ 15.png);
+}
+
+.menu-image {
+  cursor: pointer;
+  padding: 1.3em;
+}
+
+.intro {
+  width: 350px;
+  height: 100px;
+  left: 16px;
+  top: 106px;
+}
+
+.intro h1 {
+  font-size: 30px;
+  font-family: sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  color: #1a2236;
+  display: flex;
+  align-items: center;
+  padding: 0.5em 0 0 1em;
+}
+
+.summary {
+  width: 440px;
+  height: 170px;
+  left: 16px;
+  top: 100px;
+}
+
+.about-me {
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  letter-spacing: -0.01em;
+  color: #272c36;
+  margin-left: 20px;
+  margin-top: 30px;
+  margin-right: 10px;
+}
+
+.connect {
+  color: rgb(146, 146, 224);
+  margin-left: 20px;
+  padding: 10px;
+}
```

Comment:
You did it a great job changing this styling as the last code reviewer suggested however the font is different from the one provided by the Figma template. Kindly make sure that you are using the same font.  :eyes: 

![image](https://user-images.githubusercontent.com/98363075/191971024-f2410e88-268f-4c0b-be7e-651e374eeb35.png)

![image](https://user-images.githubusercontent.com/98363075/191971318-1dd0fbfd-6eda-4b26-858d-cadd70c15b00.png)